starcounter-include.html

```diff
@@ -96,6 +96,18 @@
                     }
                 });
             }
+            /**
+             * Returns host composition to clone (if have one)
+             * @readonly
+             * @type {DocumentFragment}
+             */
+            get hostComposition() {
```

Comment:
This is a very expensive function. I think it should be a method, not a getter.

Reply:
I just wanted to have it consistent with `.defaultComposition` and `.temporaryComposition`, but maybe we shouldn't have those as properties as well.